android/app/src/main/kotlin/com/jabook/app/jabook/compose/data/repository/AuthRepositoryImpl.kt

```
@@ -89,6 +90,7 @@ public class AuthRepositoryImpl
                         logger.e({ "Auth validation timeout - provider may be blocking" }, e)
                         false
                     } catch (e: Exception) {
+                        if (e is CancellationException) throw e
```

detekt (1.23.8)

[warning] Instead of catching for a general exception type and checking for a specific exception type, use multiple catch blocks.

(detekt.exceptions.InstanceOfCheckForException)